mojio_sdk/api.py

```diff
@@ -48,6 +49,28 @@ def get_vehicles(self):
         data = response.json()
         return self._parse_devices(data)
 
+    def get_trips(self):
+        if self._authentication is None or not self._authentication.is_valid():
+            self.login()
+
+        data_url = "https://" + self._tenant + "-api.moj.io/v2/trips"
+
+        auth_header = self._authentication.create_header()
+        response = requests.get(data_url, headers=auth_header)
+        data = response.json()
+        return self._parse_trips(data)
+
+    def get_trip(self, trip_id: str):
+        if self._authentication is None or not self._authentication.is_valid():
+            self.login()
+
+        data_url = "https://" + self._tenant + "-api.moj.io/v2/trips/" + trip_id
+
+        auth_header = self._authentication.create_header()
+        response = requests.get(data_url, headers=auth_header)
+        data = response.json()
```

Comment:
Basically all these lines could be a function that is re-used by the 3 other functions.